README.md

```diff
@@ -1,14 +1,48 @@
+<img src="https://raw.githubusercontent.com/ComponentDriven/csf/polish-readme/csf-header.png" width="321px" />
+
 # Storybook Component Story Format (CSF)
 
+### Why a new format?
+Components have risen to dominate the UI landscape. There are new component-oriented tools for development, testing, design, and prototyping. These tools engage in the creation and consumption of components and component examples (a.k.a. stories). But each tool has its own proprietary format because a simple, platform-agnostic way to express component examples doesn't yet exist.
+
+### The "Story" is the source of truth for a component.
+A story is a code snippet that renders an example of a component in a specific state.
+
+It uses the production code shipped to users, making it the most accurate representation of a component example. What's more, stories are expressed in the view layer you use to build your app.
+
+
+### Component Story Format
+The Component Story Format is an open standard for component examples based on JavaScript ES6 modules. This enables interoperation between development, testing, and design tools.
+
+```js
+export default { title: 'atoms/Button' };
+export const text = () => <Button>Hello</Button>;
+export const emoji = () => <Button>😀😎👍💯</Button>;
+```
+
+💎 **Simple.** Writing component "stories" is as easy as exporting ES6 functions using a clean, widely-used format.
+
+🚚 **Non-proprietary.** CSF doesn't require any vendor-specific libraries. Component stories are easily consumed anywhere ES6 modules live, including your favourite testing tools like Jest and Cypress.
+
+☝️ **Declarative.** The declarative syntax is isomorphic to higher-level formats like MDX, enabling clean, verifiable transformations.
+
+🔥 **Optimized.** Component stories don't need any libraries other than your components. And because they're ES6 modules, they're even tree-shakeable!
+
+👾 **Future-proof.** Component stories hide Storybook's underlying API, giving maintainers flexibility to improve Storybook without breaking any contracts.
+
```

Comment:
Add a section about the vendors or tools that use CSF. 

Tools: Storybook, UXPin, Webcomponents.dev, etc
Compatible with: Jest, Enzyme, Testing Library, Cypress, Mocha, etc